packages/common/src/search/_internal/hubSearchItemsHelpers/ogcItemToDiscussionPostResult.ts

```diff
@@ -0,0 +1,8 @@
+import { IOgcItem } from "./interfaces";
+import { IPost } from "../../../discussions";
+
+export async function ogcItemToDiscussionPostResult(
+  ogcItem: IOgcItem
+): Promise<IPost> {
+  return ogcItem.properties as IPost;
```

Comment:
Change to return `IHubSearchResult` - just do some quick mappings to props that make sense, and stuff the `IPost` into the `.rawResult` prop so it's available for additional manipulation when it get's dropped into the Gallery card.